test/extractor.ts

```diff
@@ -108,6 +108,7 @@ const _spmeta: string = String(readFileSync('./test/misc/spmeta.xml'));
     t.is(result.attributes.uid, 'test');
     t.is(result.attributes.mail, 'test@example.com');
     t.is(result.attributes.eduPersonAffiliation.length, 2);
+    t.deepEqual(result.attributes.missing, []);
```

Comment:
This is added to capture the existing behaviour, so we can see it change in the following commit.